adam-cli/src/main/scala/org/bdgenomics/adam/cli/Transform.scala

```diff
@@ -84,11 +84,19 @@ class TransformArgs extends Args4jBase with ADAMSaveAnyArgs with ParquetArgs {
   var forceLoadParquet: Boolean = false
   @Args4jOption(required = false, name = "-single", usage = "Saves OUTPUT as single file")
   var asSingleFile: Boolean = false
+  @Args4jOption(required = false, name = "-paired_fastq", usage = "When converting two (paired) FASTQ files to ADAM, pass the path to the second file here.")
+  var pairedFastqFile: String = null
+  @Args4jOption(required = false, name = "-record_group", usage = "Set converted FASTQs' record-group names to this value; if empty-string is passed, use the basename of the input file, minus the extension.")
+  var fastqRecordGroup: String = null
+  @Args4jOption(required = false, name = "-concat", usage = "Concatenate this file with <INPUT> and write the result to <OUTPUT>")
+  var concatFilename: String = null
```

Comment:
>> The glob syntax would result in trying to zip up one RDD … with a second

> Glob should give the union, not the zip.

Yea, but in the paired-fastq case I need something more like a `zip` (in [the validation code-path](https://github.com/bigdatagenomics/adam/pull/831/files#diff-d36ea7d0742decd0b040a73a96af06e9R387); really I just need to verify the two files' `count`s), not a `union`; my point was that, given that globbing does a `union`, it is not suitable for my use-case here.

> For me, I would see the merge command that you described as duplicative of globbing.

Only if the files being merged are all the files in a directory, right? What if I want to merge two specific files from a directory that has 10 similarly-named files in it?

The main point I'm trying to make is: we're exposing a logical "merge" capability but hiding it in the "glob" code path (which also restricts the kinds of filenames that can be merged), which feels indirect/confusing/obfuscated to me; I think it's good to support globbing but we should also expose the general capability that it represents (mergeing RDD[AR]s) more explicitly.

One semi-related thought is that maybe implying "concatenation" is not ideal, since what is really happening is a "union"? I could make this flag `-merge` instead of `-concat` if you like.

In fact, one possible approach I'd considered for exposing a more general "merge" capability was to have a `-merge` flag that takes a comma-delimited list of filenames to merge together (since args4j and the larger Scala ecosystem in 2015 have no solution for parsing a variable number of arguments, afaik).

Thanks, let me know your thoughts on all of this.